src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,30 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ *
+ * <p>The customer can have a loyalty card that has a specific number.
```

Comment:
>  that has a specific number.

These details are local to the `LoyaltyCard` type. It's what encapsulation is all about. You hide the complexity, and do not expose it all over the places in your code :)